pkg/kubectl/service_test.go

```diff
@@ -206,6 +206,7 @@ func TestGenerateService(t *testing.T) {
 							Port:       80,
 							Protocol:   "UDP",
 							TargetPort: intstr.FromString("foobar"),
+							NodePort:   80,
```

Comment:
No.  NodePorts are a different animal from service ports and are NOT taken from the main port space (default range is 30K-32K)